doc/source/examples.rst

```diff
@@ -0,0 +1,108 @@
+.. _examples:
+
+.. index:: examples
+
+Examples
+^^^^^^^^
+
+Emperor provides a Python API (:ref:`emperor-python-docs`) intended to be used
```

Comment:
The shortest path, I think, will be to use QIIME2 and its CLI. Alternatively, if we wanted, we could use the Python interface to generate the plots from within the Qiita plugin.